pandas/io/packers.py

```diff
@@ -506,11 +526,17 @@ def decode(obj):
         dtype = dtype_for(obj[u'dtype'])
         pd_dtype = pandas_dtype(dtype)
         np_dtype = pandas_dtype(dtype).base
+
+        ctor_dtype = np_dtype
+        if is_categorical_dtype(pd_dtype):
+            # Series ctor doesn't take dtype with categorical
+            ctor_dtype = None
```

Comment:
hmm, that might be a bug as in this case it is *ok* to specify the dtype (but it should be ignored). can you open a separate issue for that.